ubuntu/ubuntu-18.x/ubuntu-18.04-hpc/README.md

```diff
@@ -28,6 +29,27 @@ consistency, and reliability. This image consists of the following HPC tools and
 - Data Center GPU Manager
 - Azure HPC Diagnostics Tool
 
+This Image is compliant with the Linux Kernel 5.4.0-1043-azure.
+
 Software packages (MPI / HPC libraries) are configured as environment modules. Users can select preferred MPI or software packages as follows:
 
 `module load <package-name>`
+
+Running Single Node NCCL Test (example):
+
+```sh
+mpirun -np 8 \
+    --allow-run-as-root \
+    --map-by ppr:8:node \
+    -x LD_LIBRARY_PATH=/usr/local/nccl-rdma-sharp-plugins/lib:$LD_LIBRARY_PATH \
+    -mca coll_hcoll_enable 0 \
+    -x NCCL_IB_PCI_RELAXED_ORDERING=1 \
+    -x UCX_IB_PCI_RELAXED_ORDERING=on \
```

Comment:
pls remove 

-x UCX_IB_PCI_RELAXED_ORDERING=on
-x NCCL_NET_GDR_LEVEL=5 

If you are adding UCX_TLS=tcp, please add the interface name also.